.github/workflows/release.yml

```diff
@@ -114,12 +105,6 @@ jobs:
       # 3. push next branch
       # 4. switch back to release branch, then push
 
-      - name: "Push changes from release branch"
-        run: git push
-
-      - name: "Push release tag"
-        run: git push origin ${{ inputs.version }}
```

Comment:
I assume these two steps are handled entirely by the `git push origin --tags` line in [tag-version/action.yml](https://github.com/mongodb-labs/drivers-github-tools/blob/7edc545fcf00ff1da4910079b7488c81d7c583b0/tag-version/action.yml#L54).

Reply:
That is correct